.github/workflows/terraform-apply.yaml

```diff
@@ -0,0 +1,60 @@
+name: Apply new changes to SolEng repositories
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - terraform-plans/**
+  push:
+    branches:
+      - main
+    paths:
+      - terraform-plans/**
+
+jobs:
+  terraform:
+    name: Run Terraform
+    runs-on: ubuntu-latest
+    strategy:
+        fail-fast: false
+        matrix:
+          repository:
+            - soleng-tf-test-repo
```

Comment:
(non-blocking) This can be embedded in `tfvars` and using `for_each`, but there are pros and cons to it

Reply:
I was not working correctly for my using tfvars, but we can try it again, maybe I did something wrong.

Reply:
it depends on which var you want to iterate over; you choose to iterate over the template files, and provide repository one-by-one. Alternatively, you can choose to iterate over repositories, and define template files resource one-by-one. The latter provide a better `graph` for visualizing what will be added e.g. you can do `terraform graph | dot -Tsvg > output.svg`

Reply:
I tried to iterate over repositories, however there was an issue with usage of modules. I was not able to import staff and modules did not have access to github provider (the app credentials). Fill free to open another PR and try it, I did not manage it before Madrid

Reply:
I tried importing from root module (with my approach), and it worked fine. But I would rather not use import because it cannot address many issue (e.g. `override_on_create` is not imported correctly, pull request cannot be imported), so I would rather leave the import for github files, and have a workflow step to check if there are any update to the file. I am resolving it now, we can iterate more as we see needed